lib/resque/plugins/retry.rb

```diff
@@ -58,6 +58,8 @@ def self.extended(receiver)
       def inherited(subclass)
```

Comment:
Please update the comment on this method to indicate it is copying the callbacks as well.